lib/generators/react_on_rails/templates/client/webpack.client.base.config.js

```diff
@@ -0,0 +1,38 @@
+// https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/webpack.client.base.config.js
+// Common webpack configuration used by webpack.hot.config and webpack.rails.config.
```

Comment:
```javascript
// Common client side webpack configuration used by webpack.hot.config and webpack.rails.config.
```

Note, please apply change to source file -- you can do that on master branch for such a small change.